app/src/main/java/com/a9ts/a9ts/main/AppointmentListAdapter.kt

```diff
@@ -0,0 +1,51 @@
+package com.a9ts.a9ts.main
+
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.a9ts.a9ts.model.Appointment
+import com.a9ts.a9ts.databinding.AppointmentItemBinding
+import java.time.format.DateTimeFormatter
+import java.time.format.FormatStyle
+
+class AppointmentListAdapter(private val appointments:ArrayList<Appointment>) : RecyclerView.Adapter<AppointmentListAdapter.ViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        Log.i("AppointmentListAdapter", "ViewHolderCreated")
+
+        val binding = AppointmentItemBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return ViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+            Log.i("AppointmentListAdapter", appointments[position].toString())
+            holder.bind(appointments[position])
+        }
```

Comment:
Use Ctrl+Alt+L or option+command+L to autoformat your code. I do that hundreds of times a day. Automatically, without even noticing.